en/identity-server/next/docs/references/conditional-auth/api-reference.md

```diff
@@ -744,51 +757,93 @@ The HTTP POST function enables sending HTTP POST requests to specified endpoints
 
   - **Example**
 
   ``` js
+      var authConfig = {
+          type: "clientcredential",
+          properties: {
+              consumerKey: "clientId",
+              consumerSecret: "clientSecret",
+              tokenEndpoint: "https://token-endpoint.com/token"
+          }
+      };
+  
       function onLoginRequest(context) {
           httpPost('https://example.com/resource', {
               "email": "test@wso2.com"
           }, {
               "Authorization": "Bearer token",
               "Accept": "application/json"
-          }, {
+          }, authConfig, {
               onSuccess: function(context, data) {
-                  Log.info('httpPost call succeeded');
-                  context.selectedAcr = data.status;
+                  Log.info("Successfully invoked the external API.");
                   executeStep(1);
               },
               onFail: function(context, data) {
-                  Log.info('httpPost call failed');
-                  context.selectedAcr = 'FAILED';
+                  Log.info("Error occurred while invoking the external API.");
+                  executeStep(2);
+              },
+              onTimeout: function(context, data) {
+                  Log.info("Invoking external API timed out.");
                   executeStep(2);
               }
           });
       }
   ```
-  
+
 !!! note     
-    To restrict HTTP POST requests to certain domains, configure the `deployment.toml` file as follows:
+    To restrict HTTP GET requests to certain domains, and configure the HTTP connection settings for `httpGet` and `httpPost` functions in adaptive authentication scripts, update the `deployment.toml` file as follows:
 
-    ``` toml
+    ```toml
     [authentication.adaptive]
     http_function_allowed_domains = ["example.com", "api.example.org"]
+    
+    [authentication.adaptive.http_connections]
+    connection_timeout = 5000
+    
+    [authentication.adaptive.http_connections]
+    read_timeout = 5000
+    
+    [authentication.adaptive.http_connections]
+    request_timeout = 5000
     ```
 
+    The current default settings for timeouts and retry count are as follows:
+
+      - **HTTP_CONNECTION_TIMEOUT**: Specifies the maximum time, in milliseconds, the client will wait to establish a connection with the server.  
+        **Current value**: `3s`
+      - **HTTP_CONNECTION_REQUEST_TIMEOUT**: Defines the maximum time, in milliseconds, the client will wait to obtain a connection from the connection manager (pool).  
+        **Current value**: `1s`
+      - **HTTP_READ_TIMEOUT** (SocketTimeout): Sets the maximum time, in milliseconds, for waiting for data or, more precisely, a maximum period of inactivity between two consecutive data packets coming from the server.  
+        **Current value**: `3s`
+      - **API_REQUEST_RETRY_COUNT**: Specifies the number of retry attempts for token requests initiated for authentication from client credentials.  
+        **Current value**: `2`  
```

Comment:
Why we need to have caps here ? Why not we specify the exact config property ?

Reply:
Rather than 'Current value' use something like 'Default value'

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4772/commits/cf4fb92a77bd7856736c909b594f1ff84eb6c9a0